packages/styles/src/components/list-group.scss

```diff
@@ -1,162 +1,106 @@
 @forward './../variables/options';
-
-@use 'sass:map';
-
 @use './../themes/bootstrap/core' as *;
-@use './../themes/bootstrap/list-group' as bl;
 
-@use './../variables/color';
-@use './../variables/spacing';
-@use './../variables/type';
-@use './../variables/commons';
 @use './../variables/animation';
-@use './../variables/components/list-group';
-@use './../mixins/type' as type-mixins;
+@use './../mixins/utilities' as utilities-mx;
+
+@use '../functions/tokens';
+@use '../tokens/components';
+
+tokens.$default-map: components.$post-list;
 
 // Base class
 .list-group {
-  background-color: transparent;
+  background-color: tokens.get('list-bg');
+  list-style: none;
+  padding: 0;
 
-  &-box {
-    padding-top: map.get(spacing.$post-sizes, 'mini');
-    padding-bottom: map.get(spacing.$post-sizes, 'mini');
-    box-shadow: commons.$box-shadow;
+  > li {
+    border: unset;
+    padding: 0;
   }
 }
 
-.list-group-item-action {
-  @include transition(
-    color animation.$transition-time-simple animation.$transition-easing-default,
-    background-color animation.$transition-time-simple animation.$transition-easing-default
-  );
-
-  &:hover,
-  &:focus {
-    .list-group-item-right-svg {
-      transform: translateX(animation.$transition-distance-xsmall);
-    }
+.list-group-link,
+.list-group-document,
+.list-group-switch {
+  display: flex;
+  position: relative;
+  padding-block-start: tokens.get('list-item-padding-block-start');
+  padding-block-end: tokens.get('list-item-padding-block-end');
+  padding-inline-start: tokens.get('list-item-padding-inline-start');
+  padding-inline-end: tokens.get('list-item-padding-inline-end');
+  border-radius: tokens.get('list-item-focus-outline-radius'); // Used for the focus only
+  color: tokens.get('list-item-enabled-fg');
+
+  &::after {
+    display: block;
+    content: '';
+    position: absolute;
+    inset: auto 0 0;
+    border-block-end: tokens.get('list-item-border-width') tokens.get('list-item-border-style')
+      tokens.get('list-item-enabled-border');
   }
 
-  .list-group-item-right-svg {
-    @include transition(
-      transform animation.$transition-time-area-small animation.$transition-easing-default
-    );
-  }
+  &:hover {
+    color: tokens.get('list-item-hover-fg');
 
-  &-minimal,
-  &-transparent {
-    &:hover,
-    &:focus {
-      background-color: transparent;
+    &::after {
+      border-block-end-color: tokens.get('list-item-hover-border');
     }
   }
 
-  &-inverted {
-    background-color: transparent;
-
-    &:hover,
-    &:focus {
-      background-color: list-group.$list-group-hover-bg-inverted;
-    }
-  }
+  @include utilities-mx.focus-style(
+    $offset: tokens.get('list-item-focus-outline-offset'),
+    $width: tokens.get('list-item-focus-outline-width'),
+    $color: tokens.get('list-item-focus-outline-color'),
+    $style: tokens.get('list-item-focus-outline-style')
```

Comment:
I think you will need to remove these tokens, as they do not exist on `main` anymore.
The general focus tokens are now used by the focus mixins itself.